compose/queue/api_data/taskqueue_v1beta2.api

```diff
@@ -0,0 +1,528 @@
+{
+ "kind": "discovery#restDescription",
+ "etag": "\"YWOzh2SDasdU84ArJnpYek-OMdg/rBU1z2tHcOAAx8ktsJNBvusBoYw\"",
```

Comment:
An etag is a fingerprint, for quick comparison. I should probably change it though.